content/en/blog/_posts/2022-10-27-live-and-let-live-with-kluctl-and-ssa.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog
+title: "Live and let live with Kluctl and Server Side Apply"
+date: 2022-10-27
+slug: live-and-let-live-with-kluctl-and-ssa
+---
+
+**Author:** Alexander Block (@codablock)
+
+This blog post was inspired by a previous Kubernetes blog post about
+[Advanced Server Side Apply](https://kubernetes.io/blog/2022/10/20/advanced-server-side-apply/).
+The author of this blog post listed multiple benefits for applications and
+controllers when switching to Server-side apply. Especially the chapter about
+[CI/CD systems](https://kubernetes.io/blog/2022/10/20/advanced-server-side-apply/#ci-cd-systems)
+motivated me to respond and write down my thoughts and experiences.
+
+These thoughts and experiences are the results of me working on [Kluctl](https://kluctl.io)
+for the past 2 years. I describe Kluctl as "The missing glue to put together
+large Kubernetes deployments, composed of multiple smaller parts
+(Helm/Kustomize/...) in a manageable and unified way."
+
+To get a basic understanding of Kluctl, I suggest to visit the [kluctl.io](https://kluctlio)
+website and read through the documentation and tutorials, for example the 
+[microservices demo tutorial](https://kluctl.io/docs/guides/tutorials/microservices-demo/).
+As an alternative, you can watch [Hands-on Introduction to kluctl](https://www.youtube.com/watch?v=9LoYLjDjOdg)
+from the Rawkode Academy YouTube channel which shows a hands-on demo session.
+
+There is also a [Kluctl delivery scenario](https://github.com/podtato-head/podtato-head/tree/main/delivery/kluctl)
+available in the [podtato-head](https://github.com/podtato-head/podtato-head) demo project.
+
+## Live and let live
+
+One of the main philosophies that Kluctl follows is ["live and let live"](https://kluctl.io/docs/philosophy/#live-and-let-live),
+meaning that it will try its best to work in conjunction with any other tool or
+controller running outside or inside your clusters. Kluctl will not overwrite
+any fields that it lost ownership of, unless you explicitly tell it to do so.
+
+Achieving this would not have been possible (or at least be several magnitudes
+harder) without the use of Server-side apply. Server-side apply allows Kluctl
+to detect when ownership for a field got lost, for example when another controller
+or operator updates that field to another value. Kluctl can then decide on a 
+field-by-field basis if a force-apply is required and then retry based on these
```

Comment:
Omit `and then` to avoid duplication with `can then` the line before. I'd suggest `[...] before retrying based on these decisions`